src/rest/rest-api-service.js

```diff
@@ -0,0 +1,22 @@
+import fetch from 'node-fetch';
+import ApiResponse from './api-response.js';
+
+class RestApiService {
+  constructor(baseUrl) {
+    this.baseUrl = baseUrl;
+  }
+
+  fetch(method, endpoint) {
+    return fetch(`${this.baseUrl}${endpoint}`, { method }).then(response => {
```

Comment:
Maybe add some spaces to improve readability

Reply:
If prettier didn't fix it, then there is not point to do so. Just run npm run prettier to be sure.